filament/src/Renderer.cpp

```diff
@@ -921,10 +952,6 @@ bool FRenderer::beginFrame(FSwapChain* swapChain, uint64_t vsyncSteadyClockTimeN
     auto beginFrameInternal = [=]() {
         FEngine& engine = getEngine();
         FEngine::DriverApi& driver = engine.getDriverApi();
-
-        if (callback) {
-            driver.setFrameScheduledCallback(swapChain->getHwHandle(), callback, user);
```

Comment:
We still need to support this somehow.

Reply:
But that API was marked deprecated. Do we need to keep it?

Reply:
I'm fine with getting rid of it, but I think that would constitute an API breaking change. I'm not sure if any Google clients are still using it, but it should be easy to migrate them.